### ISSUE TITLE
No module named _curses

### ISSUE THREAD

**Issue by abdilahrf** [Gallopsled/pwntools#572]
```
E:\Untitled Folder>python test.py
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from pwn import *
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwn\__init__.py",
 line 2, in <module>
    from .toplevel import *
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwn\toplevel.py",
 line 15, in <module>
    import pwnlib
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwnlib\__init__.p
y", line 35, in <module>
    importlib.import_module('.%s' % module, 'pwnlib')
  File "C:\Python27\lib\importlib\__init__.py", line 37, in import_module
    __import__(name)
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwnlib\asm.py", l
ine 57, in <module>
    from .log import getLogger
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwnlib\log.py", l
ine 114, in <module>
    from . import term
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwnlib\term\__ini
t__.py", line 3, in <module>
    from . import key
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwnlib\term\key.p
y", line 10, in <module>
    from . import termcap
  File "C:\Python27\lib\site-packages\pwntools-2.2.0-py2.7.egg\pwnlib\term\termc
ap.py", line 2, in <module>
    import curses
  File "C:\Python27\lib\curses\__init__.py", line 15, in <module>
    from _curses import *
ImportError: No module named _curses
```

i got this error why this happening 
but all dependencies succesfully installed from github latest commit

**Comment by zachriggle** [comment]
Pwntools does not support Windows.  Use a real OS ;)

**Comment by br0ns** [comment]
TL;DR: Won't fix, closing.

Ideally we would just parse the terminfo database ourselves (and use a sane default if it doesn't exist).

But that is another problem; the real problem here is that `pwnlib` doesn't work on Windows.  And I don't see that happening anytime soon.  Or ever for that matter.

If you won't give up Windows I would suggest running Linux in a VM.
